app/views/api/v2/user_columns/main.json.rabl

```diff
@@ -0,0 +1,5 @@
+object @user_column
+attributes :id, :name, :columns, :created_at, :updated_at
+child :user do
```

Comment:
Are you sure we need to include the user in the user_column view?

Reply:
have no issues taking this out